Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.h

```diff
@@ -130,4 +130,5 @@ class CLuaWorldDefs : public CLuaDefs
     static std::variant<bool, float, CLuaMultiReturn<float, float, float>> GetWorldProperty(eWorldProperty property);
     static bool SetWorldProperty(eWorldProperty property, float arg1, std::optional<float> arg2, std::optional<float> arg3);
     static bool ResetWorldProperty(eWorldProperty property);
+    static bool SetVolumetricShadowsEnabled(bool bEnable);
```

Comment:
We avoid using Hungarian notation in new codes, please rename it simply to `enable`